pwa-worker.js

```diff
@@ -1,3 +1,5 @@
+import logger from './react/features/base/logging/logger';
```

Comment:
Is there such a thing?

Reply:
Not sure this is correct at all ... have you tested this at all?
Thie is a service worker and is run outside of the app and is deliberately using console logging. Logger depends on configuration which is not enabled and available here.

Reply:
What problem are you trying to solve?

Reply:
Well what I mean, is to fix the issues you have experienced and find problematic.

Reply:
The PWA worker is outside of the main jitsi meet app, so it doesn't follow the same rules. Please leave it out.